<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenControlWidget</class>
 <widget class="QWidget" name="ScreenControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="OrganizationTree" name="orgTreewidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <item>
      <widget class="ToolBarWidget" name="windowToolBar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PlayerLayout" name="screenLayout" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayerLayout</class>
   <extends>QWidget</extends>
   <header>layout/playerlayout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrganizationTree</class>
   <extends>QWidget</extends>
   <header>organizationtree.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolBarWidget</class>
   <extends>QWidget</extends>
   <header>toolbarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
